### ISSUE TITLE
Package missing on doing yarn install

### ISSUE THREAD

**Issue by riginoommen** [spotify/backstage#643]
## Expected Behavior

``` Yarn install ``` process should be successfully executed for the dependency installs
## Current Behavior

```bash
riginoommen-mac:study riginoommen$ git clone https://github.com/spotify/backstage.git
Cloning into 'backstage'...
remote: Enumerating objects: 341, done.
remote: Counting objects: 100% (341/341), done.
remote: Compressing objects: 100% (224/224), done.
Receiving objects:  57% (6990/12256), 11.86 MiB | 810.00 KiB/s 
Receiving objects:  94% (11595/12256), 15.67 MiB | 725.00 KiB/s
remote: Total 12256 (delta 168), reused 233 (delta 111), pack-reused 11915
Receiving objects: 100% (12256/12256), 22.37 MiB | 1.05 MiB/s, done.
Resolving deltas: 100% (6929/6929), done.
riginoommen-mac:study riginoommen$ 
riginoommen-mac:study riginoommen$ 
riginoommen-mac:study riginoommen$ cd backstage/
riginoommen-mac:backstage riginoommen$ yarn install
yarn install v1.22.1
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
error Couldn't find package "@types/react-helmet@^5.0.15" required by "@backstage/core@0.1.1-alpha.4" on the "npm" registry.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
riginoommen-mac:backstage riginoommen$ node -v
v12.16.1
```

Currently dependency installation fails and terminates the process cycle

## Possible Solution
May need to update the `package.json`

## Steps to Reproduce

1. Clone the repo
2. execute ```yarn install```

## Your Environment

- NodeJS Version (v12): v12.16.1
- Operating System and Version (e.g. Ubuntu 14.04): macOS Catlina 10.15.4
- Browser Information: Google Chrome

**Comment by freben** [comment]
Hm! That's odd, that package seems to be published 4 months ago:

https://www.npmjs.com/package/@types/react-helmet

If you make a new empty directory somewhere, and then do `yarn init` followed by `yarn add --dev @types/react-helmet`, what does it do?

**Comment by riginoommen** [comment]
Hi @freben that worked sending PR in a moment

**Comment by Rugvip** [comment]
Expecting this to have been fixed by #645
